libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -150,6 +150,7 @@ IotI2CHandle_t iot_i2c_open( int32_t lI2CInstance );
 /**
  * @brief Sets the application callback to be called on completion of an operation.
  *
+ * @note This is only used in asynchronous call.
```

Comment:
I think we need to indicate that only 1 callback is supported.  So, if this is called twice, the previous callback function will be over-written.  Separate callback functions are read_async and write_async are not supported.

Reply:
Good point. I will clarify